index.html

```diff
@@ -0,0 +1,144 @@
+                <p class="section-description">Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit.</p>
+                <div class="categories">
+                    <a href="##" class="categories__item active_item" id="all">All</a>
+                    <a href="##" class="categories__item" id="web_design" role="button">Web Design</a>
+                    <a href="##" class="categories__item" id="graphic_design" role="button">Graphic Design</a>
+                    <a href="##" class="categories__item" id="artwork" role="button">Artwork</a>
+                </div>
+            </div>
+            <div class="portfolio__content" id="shuffle">
+                <img src="img/Portfolio/portfolio-1.png" alt="Portfolio expample picture"  class="content-picture" id="img">
+                <img src="img/Portfolio/portfolio-2.png" alt="Portfolio expample picture"  class="content-picture" id="img">
+                <img src="img/Portfolio/portfolio-3.png" alt="Portfolio expample picture"  class="content-picture" id="img">
+                <img src="img/Portfolio/portfolio-4.png" alt="Portfolio expample picture"  class="content-picture" id="img">
+                <img src="img/Portfolio/portfolio-5.png" alt="Portfolio expample picture"  class="content-picture" id="img">
+                <img src="img/Portfolio/portfolio-6.png" alt="Portfolio expample picture"  class="content-picture" id="img">
+                <img src="img/Portfolio/portfolio-7.png" alt="Portfolio expample picture"  class="content-picture" id="img">
+                <img src="img/Portfolio/portfolio-8.png" alt="Portfolio expample picture"  class="content-picture" id="img">
+                <img src="img/Portfolio/portfolio-9.png" alt="Portfolio expample picture"  class="content-picture" id="img">
+                <img src="img/Portfolio/portfolio-10.png" alt="Portfolio expample picture"  class="content-picture" id="img">
+                <img src="img/Portfolio/portfolio-11.png" alt="Portfolio expample picture"  class="content-picture" id="img">
+                <img src="img/Portfolio/portfolio-12.png" alt="Portfolio expample picture"  class="content-picture" id="img">
+            </div>
+        </section>
+    </main>
+    <hr class="dividing-line">
+    <footer class="footer main-section" id="footer">
+        <p class="copyright">© Copyright 2020</p>
+        <div class="contacts">
+            <a href="##" class="contact__link"><img src="img/Contacts/ico-facebook.svg" alt="facebook icon" class="contacts__icon"></a>
+            <a href="##" class="contact__link"><img src="img/Contacts/ico-google-plus.svg" alt="google plus icon" class="contacts__icon"></a>
+            <a href="##" class="contact__link"><img src="img/Contacts/ico-twitter.svg" alt="twitter icon" class="contacts__icon"></a>
+            <a href="##" class="contact__link"><img src="img/Contacts/ico-linkedin.svg" alt="linkedin icon" class="contacts__icon"></a>
```

Comment:
usually such icons are not inserted as images but as glyphs from  [fonawesome](https://fontawesome.com/icons?d=gallery)